packages/font-glyphs/src/letter/latin/c.ptl

```diff
@@ -414,7 +414,5 @@ glyph-block Letter-Latin-C : begin
 
 	derive-composites 'CCedilla' 0xC7 'C' 'cedillaExtShapeBelowOArc'
 	derive-composites 'cCedilla' 0xE7 'c' 'cedillaExtShapeBelowOArc'
-	derive-composites 'CCedillaAcute' 0x1E08 'C' 'cedillaExtShapeBelowOArc' 'acuteAbove'
-	derive-composites 'cCedillaAcute' 0x1E09 'c' 'cedillaExtShapeBelowOArc' 'acuteAbove'
-	derive-composites 'cyrl/The' 0x4AA 'cyrl/Es' 'cedillaExtShapeBelowOArc'
-	derive-composites 'cyrl/the' 0x4AB 'cyrl/es' 'cedillaExtShapeBelowOArc'
+	derive-composites 'cyrl/The' 0x4AA 'cyrl/Es' 'ogonekBelow'
```

Comment:
Please revert to use Cedilla. The Es-Ogonek form is only considered as "Occasionally used" per Wikipedia.